packages/lib/slots.test.ts

```diff
@@ -296,8 +296,9 @@ describe("Tests the slot logic", () => {
       ],
     });
 
-    expect(slots).toHaveLength(1);
-    expect(slots[0].time.format()).toBe("2023-07-13T08:00:00+05:30");
+    expect(slots).toHaveLength(2);
+    expect(slots[0].time.format()).toBe("2023-07-13T07:30:00+05:30");
+    expect(slots[1].time.format()).toBe("2023-07-13T08:30:00+05:30");
```

Comment:
For this test with given inputs, i guess its correct to have 2 slots available from 7:30 to 8:30 with eventLength of 60mins